src/storybook/decorators.tsx

```diff
@@ -28,7 +28,7 @@ export const ThemeToggler = (DecoratedStory: () => JSX.Element): JSX.Element =>
   return (
     <>
       <ThemeProvider theme={theme}>
-        <Frame style={{ backgroundColor: darkMode ? COLOURS.bgDark : COLOURS.bgLight }}>{DecoratedStory()}</Frame>
+        <Frame style={{ background: darkMode ? COLOURS.bgDark : COLOURS.bgLight }}>{DecoratedStory()}</Frame>
```

Comment:
Is it an idea to further abstract this away into a 'setMode' type of hook? So that the hook determines what mode we're on and return either `COLOURS.bgDark` or `COLOURS.bgLight` ?

Down the road perhaps we can even further abstract it down to:
`styles.backgroundDefault` which first determines what mode we're on and then only return the bg color.

Reply:
> Down the road perhaps we can even further abstract it down to:
styles.backgroundDefault which first conditionally determines what mode we're on and then only return the bg color.

how would you do this? `styles` is in another file and should remain there. To conditionally set it there wouldn't be possible unless it used some logic elsewhere or within a component. 

The hook approach is imo the only viable solution, or at least the most straight forward without spending too much time. The user's selection would then be saved to localstorage and loaded on app load (as we do now in v1)

Reply:
for now this is just a lazy man's version of creating a decorator i needed for testing, it will be extrapolated into a hook down the line (sth to check in v1 if we already have, which i believe we do and can repurpose)

Reply:
Yeah was thinking something along the lines of a hook. Assuming we can call that from within the styles file context. So styles would only return you stuff, _after_ checking the current mode with hooks.